.github/ISSUE_TEMPLATE/frontend_report.yml

```
@@ -0,0 +1,75 @@
+name: "🌐 Frontend (Issue / Optimization / Documentation)"
+description: Report an issue specifically related to the website frontend. For other inquiries, please use the Discussions section.
+title: "[Frontend]: "
+labels: "Status: Unconfirmed"
```

```suggestion
labels: "website"
```
if we use the website label we don't need the verbose title (which user also can remove in edit)